apps/playground-sveltekit/.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
We already have Prettier set up for the monorepo, let's keep it consistent. Please remove this file and run prettier on the affected files to apply a consistent linting.

Reply:
Fixed. The config matched so `format` didn't create any changes.